react/private/Icon/Icon.less

```diff
@@ -9,3 +9,47 @@
   width: 16px;
   height: 16px;
 }
+
+.heroSvg {
+  .iconSize(
+    @typography-scale: @hero-type-scale,
+    @typography-scale-mobile: @hero-type-scale-mobile
+  );
+}
+
+.headlineSvg {
+  .iconSize(
+    @typography-scale: @headline-type-scale,
+    @typography-scale-mobile: @headline-type-scale-mobile
+  );
+}
+
+.headingSvg {
+  .iconSize(
+    @typography-scale: @heading-type-scale,
+    @typography-scale-mobile: @heading-type-scale-mobile
+  );
+}
+
+.superstandardSvg {
+  .iconSize(@superstandard-type-scale);
+}
+
+.standardSvg {
+  .iconSize(
+    @typography-scale: @standard-type-scale,
+    @typography-scale-mobile: @standard-type-scale-mobile
+  );
+}
+
+.substandardSvg {
+  .iconSize(@substandard-type-scale);
+}
+
+.smallSvg {
+  .iconSize(@small-type-scale);
+}
+
+.interactionSvg {
```

Comment:
This makes me wonder whether we need to make `interaction` more of an official size, so `<Text size="interaction">` works, otherwise you have to remember which components support it and which don't.

Again, don't have to do this right now, just raising the question.

Reply:
Yeah, I was thinking about that too, I think we need to add it

Reply:
Are we adding it in this PR? What do you think?